rct229/ruleset_functions/get_primary_secondary_loop_dict.py

```diff
@@ -0,0 +1,113 @@
+from itertools import chain
+
+from rct229.data.schema_enums import schema_enums
+from rct229.ruleset_functions.baseline_systems.baseline_system_util import (
+    find_exactly_one_fluid_loop,
+)
+from rct229.utils.assertions import getattr_
+from rct229.utils.jsonpath_utils import find_all
+
+FLUID_LOOP = schema_enums["FluidLoopOptions"]
+
+APPLICABLE_SYS_TYPES = [
```

Comment:
@weilixu  The RDS includes systems 11B and 13B in the applicable systems.  But neither of these appear in the enum in `baseline_system_util.py`.  Should I add these to the enumeration?

Reply:
The RDS is quite messy regarding the HVAC system type.
The 11B is changed to 11.1B, and 13B is no longer exist. It will be great if you can update the RDS to keep it consistent. I will check again on the baseline system types function and make sure all the systems are correct and consistent with their sub-functions.